### DISCUSSION TITLE
check Point Mobile Access Portal Agent internal error (code 50)

### DISCUSSION THREAD

**Discussion by Jhonarrr** [ruyrybeyro/chrootvpn#34] [General]
i jhave this error
![Screenshot 2024-07-15 at 22-47-33 Check Point Mobile - Sign In](https://github.com/user-attachments/assets/667c264c-9a92-4fea-b702-9c1375975eab)

**Comment by ruyrybeyro** [comment]
Hi, 

Per the error that is a compliance security check defined by your organization.  You can connect only from desktop devices approved by them, and/or obeying pre defined security requirements.

Regards
